src/likes/likes.service.ts

```diff
@@ -35,7 +39,8 @@ export class LikesService {
                 postId
             }
         });
-        return likePost;
+        const action = await this.actionsService.addAction('Create',userId,'PostLike',likePost.id,likePost)
```

Comment:
here you have hadcoded strings. Store them in variables and reuse? insted of hardcode them